pkg/api/vlabs/validate.go

```diff
@@ -1474,6 +1473,30 @@ func (a *Properties) validateContainerRuntime() error {
 	return nil
 }
 
+func (a *Properties) isValidVersion(k8sVersion string) (bool, error) {
```

Comment:
this function should probably move to pkg/api/common/versions.go

Reply:
thanks, moved to `pkg/api/common/versions.go`